docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -48,16 +51,45 @@ and the following line to `spark/conf/spark-env.sh`:
 
 This section shows how to use Alluxio as input and output sources for your Spark applications.
 
-Put a file `foo` into HDFS, assuming namenode is running on `localhost`:
+### Use Data Already in Alluxio
+
+First, we will copy some local data to the Alluxio file system. Put the file `LICENSE` into Alluxio,
+assuming you are in the Alluxio project directory:
 
-{% include Running-Spark-on-Alluxio/foo.md %}
+{% include Running-Spark-on-Alluxio/license-local.md %}
 
 Run the following commands from `spark-shell`, assuming Alluxio Master is running on `localhost`:
 
-{% include Running-Spark-on-Alluxio/Alluxio-In-Out-Scala.md %}
+{% include Running-Spark-on-Alluxio/alluxio-local-in-out-scala.md %}
+
+Open your browser and check [http://localhost:19999](http://localhost:19999). There should be an
```

Comment:
this could be `localhost:19999/browse` so that they don't have to find the tab themselves